sensor_msgs/Image depth_image
sensor_msgs/Image segmask
sensor_msgs/CameraInfo cam_info
geometry_msgs/PoseStamped cam_pose
MapInfo map_info
float32[] goal_pose  # (x, y, theta)
---
nav_msgs/Path path
# ContactPoint[] push_contact